examples/tests/live_monitoring_README.md

```diff
@@ -0,0 +1,4 @@
+To test the online monitoring feature, 
```

Comment:
For testing, I think we should be using the Jasmine + Karma framework I set up. This way we can run automated tests, instead of needing to do them manually. Angular has ways of mocking up asynchronous calls to mock calls to a remote server. If you are not familiar with these testing frameworks, let me know, and I can work on it.